Procfile

```diff
@@ -0,0 +1,3 @@
+# Run Rails & Webpack concurrently
+rails: bundle exec bin/rails server -b 0.0.0.0
```

Comment:
it could be something like  (not tested)
```sh
export RAILS_STARTUP='puma -w 3 --preload' 
foreman start
```
then in your code do something like

```sh
rails:  [ -z "$RAILS_STARTUP" ] && $RAILS_STARTUP || bin/rails server -b 0.0.0.0;
```